app/components/common/Navigation.vue

```diff
@@ -449,15 +449,16 @@ export default Vue.extend({
               span.icon-bar
               span.icon-bar
             .navbar-brand
-              a(:href="homeLink")
-                picture(v-if="!me.showChinaResourceInfo()")
-                  source#logo-img.powered-by(srcset="/images/pages/base/logo.webp" type="image/webp")
-                  img#logo-img.powered-by(src="/images/pages/base/logo.png" alt="CodeCombat logo")
-                img(v-else src="/images/pages/base/logo-cn.png" alt="CodeCombat logo")
+              a(v-if="isOzaria" :href="homeLink")
+                img#logo-img.oz-logo(src="/images/ozaria/home/ozaria_home_logo.png" alt="Ozaria logo" title='Ozaria')
```

Comment:
let's convert the image to webp format. It is widely used now and takes much less size. You can use an online tool to convert the ozaria_home_logo.png to webp format